core/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -484,7 +485,7 @@ public VersionType versionType() {
     }
 
 
-    public void process(@Nullable MappingMetaData mappingMd, String concreteIndex) {
+    public void process(@Nullable Version indexCreatedVersion, @Nullable MappingMetaData mappingMd, String concreteIndex) {
```

Comment:
I wonder, given my comment about the null index metadata if this should not be nullable?